### PR TITLE
russian translation cleanup

### PR DESCRIPTION
**Pull request by JinEnMok**
Fixed a few errors in the existing translation and added a few to items that were previously not translated. 

There are still things that are not translated, e.g. the "width" and "height" fields, some buttons in the keyboard shortcut submenu, the right-click menu, etc.

### CODE REVIEW

File: translations/ru.po [hunk 1]
```diff
@@ -87,7 +87,7 @@ msgstr "Подвинуть вниз"
 
 #: src/ui_elements/palette_config.gd
 msgid "Copy as XML"
-msgstr "Скопировать как XML"
+msgstr "Скопировать XML"
```
Comment (MewPurPur) [inline translations/ru.po:90]: Are you sure about this? The context for this is, you're converting a ColorPalette resource into XML and copying that, so you can easily share it around. There's no XML until you "copy as XML".

File: translations/ru.po [hunk 2]
```diff
@@ -239,27 +239,27 @@ msgstr "Внешний вид"
 
 #: src/ui_parts/display.gd src/TranslationUtils.gd
 msgid "Load reference image"
-msgstr ""
+msgstr "Загрузить "
```
Comment (MewPurPur) [inline translations/ru.po:242]: I think this is incomplete?

File: translations/ru.po [hunk 3]
```diff
@@ -3,25 +3,25 @@ msgid ""
...
 "Language: ru\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 3.4.2\n"
+"X-Generator: Lokalize 24.05.0\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
 msgid "translation-credits"
 msgstr "volkov <volkovissocool@gmail.com>"
```
Comment (MewPurPur) [inline translations/ru.po:16]: Add yourself if you want to be credited

Comment (JinEnMok) [reply translations/ru.po:16]: do I add another msgid/msgstr or do i append myself to the existing msgstr?

File: translations/ru.po (hunk 2 above)
Comment (JinEnMok) [reply translations/ru.po:242]: Good catch, cheers

File: translations/ru.po (hunk 3 above)
Comment (MewPurPur) [reply translations/ru.po:16]: I don't remember, look at the German localization file.

**Comment by MewPurPur** [comment]
I'm preparing alpha 4 right now, but if you're quick, I can sneak this in just before I bump the version and cease all development until the release. (* ^ ω ^)

**Comment by JinEnMok** [comment]
Should be done now. Will look into translating non-translated fields for the next release then

**Comment by MewPurPur** [comment]
Thanks!

**Pull request merged**